plugins/web/include/web/configuration.hpp

```diff
@@ -19,9 +19,13 @@ namespace vast::plugins::web {
 struct configuration {
   static const record_type& schema() noexcept {
     static auto result = vast::record_type{
-      {"bind", vast::string_type{}},    {"port", vast::int64_type{}},
-      {"mode", vast::string_type{}},    {"certfile", vast::string_type{}},
-      {"keyfile", vast::string_type{}}, {"web-root", vast::string_type{}},
+      {"bind", vast::string_type{}},
+      {"port", vast::int64_type{}},
+      {"mode", vast::string_type{}},
+      {"certfile", vast::string_type{}},
+      {"keyfile", vast::string_type{}},
+      {"web-root", vast::string_type{}},
+      {"cors-allowed-origin", vast::string_type{}},
```

Comment:
This new argument must be documented; it also requires a changelog entry announcing its existence.